tests/testthat/test-robustbayesianmetaanalysis.R

```diff
@@ -1067,7 +1067,7 @@ fittedPath <- file.path("robmaFit.RDS")
   options$resultsIndividualSingleNumber <- 12
   options$resultsModelsBF <- "inclusion"
   options$resultsModelsOrder <- "default"
-  options$resultsTheta <- TRUE
+  options$resultsTheta <- FALSE
```

Comment:
why? what changed? can we just temporarily skip the failing tests?

Reply:
The package uses different parametrization of the likelihood so it does not estimate the Theta coefficients anymore. I kept some functionality associated with their plotting and etc in the JASPs backend I manage to add them back but they are not available to users anymore and the error would throw an error if they were requested.